shoop/core/models/_contacts.py

```diff
@@ -150,10 +172,16 @@ def is_all_seeing(self):
 class AnonymousContact(Contact):
     pk = id = None
     is_anonymous = True
+    default_contact_group_identifier = "default_anonymous_group"
+    default_contact_group_name = _("Anonymous Contacts")
 
     class Meta:
         managed = False  # This isn't something that should actually exist in the database
 
+    def __init__(self, *args, **kwargs):
+        self.get_default_contact_group()  # Make sure default anonymous contact group is created
```

Comment:
It seems somewhat scary that an `__init__` would have the side effect of creating an unrelated object in the database.

Would it be better to do this in `.groups`